### ISSUE TITLE
Madagascar: Import new HASC codes for admin1 and admin2 features

### ISSUE THREAD

**Issue by stepps00** [whosonfirst-data/whosonfirst-data#616]
In October 2009, the six existing `admin1` regions in Madagascar were dissolved into 22 new `admin1` regions. Additionally, 119 `admin2` features were formed. Who's On First classifies these features correctly, but Statoids is out of date and does not have correct HASC concordance for administrative features in Madagascar.

Once new features are imported from mesoshapes, add in a HASC field to port over additional properties. Also, since the six original regions were dissolved into 22 new regions without any border changes (aside from division within the region itself), it is possible to import the older regions with superseding/cessation date work completed.

**Comment by stepps00** [comment]
Features at the `region` level have been updated with HASC codes in [this commit](https://github.com/whosonfirst-data/whosonfirst-data/commit/463c22c3c0df4b69d77255551c163b0492d59d49).

**Comment by stepps00** [comment]
Update for Madagascar:

- Statoids `admin1` divisions no longer exist as of 2007 ([source]())
- Statoids `admin2` divisions are currently the primary division in Madagascar, WOF correctly has these classified as regions ([wof records](https://whosonfirst.mapzen.com/spelunker/id/85632223/descendants/?exclude=nullisland&placetype=region)). These WOF records have been updated in [this commit](https://github.com/whosonfirst-data/whosonfirst-data/commit/be01c4bd3eb3067fcb8cc3d756409f250ed58049) with Statoids data and new hasc codes. Once that commit is merged, close this issue.
- WOF `county` records do not line up with anything in Statoids. No work to do at this placetype.

**Comment by stepps00** [comment]
All of the referenced issues above have been merged/closed, closing issue.

tl;dr: Statoids maintains out of date Madagascar data. The [region (`admin1`) records in WOF](https://whosonfirst.mapzen.com/spelunker/id/85632223/descendants/?exclude=nullisland&placetype=region) have `hasc:id` concordance values. County (`admin2`) records do not have `hasc:id` concordances, as Statoids does not maintain them.
